vorestation.dme

```diff
@@ -1164,6 +1165,7 @@
 #include "code\game\objects\items\devices\chameleonproj.dm"
 #include "code\game\objects\items\devices\debugger.dm"
 #include "code\game\objects\items\devices\defib.dm"
+#include "code\game\objects\items\devices\denecrotizer.dm"
```

Comment:
This should probably have _vr at the end so we know it is a vorestation-specific file.